CleverPush/Source/CPAppBannerModuleInstance.m

```diff
@@ -196,7 +196,17 @@ - (void)getBanners:(NSString*)channelId bannerId:(NSString*)bannerId notificatio
 
     NSMutableURLRequest* request = [[CleverPushHTTPClient sharedClient] requestWithMethod:HTTP_GET path:bannersPath];
     [CleverPush enqueueRequest:request onSuccess:^(NSDictionary* result) {
-        NSArray *jsonBanners = [result objectForKey:@"banners"];
+        NSMutableArray *jsonBanners = [[NSMutableArray alloc]init];
+        if(categoryId != nil && ![categoryId isEqualToString:@""]) {
+            for(NSDictionary *json in [result objectForKey:@"banners"]) {
+                if([json objectForKey:@"categoryID"] != nil && [[json objectForKey:@"categoryID"] isEqualToString:categoryId]) {
```

Comment:
categoryID -> category

Reply:
@gbhrdt , "**categoryID**" is the key, which will come from the backend (API's response), but okay, let me change it to **category** from **categoryID**.